android/app/src/googleplay/java/org/fossasia/openevent/fragments/MapsFragment.java

```diff
@@ -168,52 +190,9 @@ private BitmapDescriptor vectorToBitmap(Context context, @DrawableRes int id, in
         return BitmapDescriptorFactory.fromBitmap(bitmap);
     }
 
-    private void handleSearchEvents() {
-        ArrayAdapter<String> adapter = new ArrayAdapter<>(getActivity(),
-                android.R.layout.simple_spinner_dropdown_item, searchItems);
-        searchView.setAdapter(adapter);
-
-        searchView.setOnClickListener(v -> searchView.setText(""));
-
-        searchView.setOnItemClickListener((parent, view, position, id) -> {
-            View scene = getActivity().getCurrentFocus();
-            if (scene != null) {
-                InputMethodManager imm = (InputMethodManager) getActivity()
-                        .getSystemService(Context.INPUT_METHOD_SERVICE);
-                imm.hideSoftInputFromWindow(scene.getWindowToken(), 0);
-            }
-            String s = parent.getItemAtPosition(position).toString();
-            int pos=searchItems.indexOf(s);
-            LatLng lng = new LatLng(mLocations.get(pos).getLatitude(), mLocations.get(pos).getLongitude());
-            mMap.animateCamera(CameraUpdateFactory.newLatLngZoom(lng,
-                    (float) Math.floor(mMap.getCameraPosition().zoom + 8)));
-        });
-    }
-
-    @Override
-    public void onCreateOptionsMenu(Menu menu, MenuInflater inflater) {
-        menu.clear();
-        inflater.inflate(R.menu.menu_map, menu);
-    }
-
-    @Override
-    public boolean onOptionsItemSelected(MenuItem item) {
-        switch (item.getItemId()) {
-            case R.id.share_map_url:
```

Comment:
@rohanagarwal94 There is a separate issue for it, to align with the droid version better.